src/scanpy/preprocessing/_pca/__init__.py

```diff
@@ -44,12 +44,14 @@
 SvdSolvTruncatedSVDDaskML = Literal["tsqr", "randomized"]
 SvdSolvDaskML = SvdSolvPCADaskML | SvdSolvTruncatedSVDDaskML
 
-SvdSolvPCASklearn = Literal["auto", "full", "arpack", "randomized"]
+SvdSolvPCASklearn = Literal["auto", "full", "arpack", "covariance_eigh", "randomized"]
```

Comment:
Are we saying then that we support this in this PR? Maybe a different PR for this?  Not sure what the protocol for this sort of thing is

Reply:
i.e., scipy sparse without dask + covariance_eigh?

Reply:
OK, extracted to #3296